qubes/tests/qmemman.py

```diff
@@ -24,27 +24,27 @@
 import qubes.tests
 
 
-def MB(val):
+def in_megabyte(val):
```

Comment:
The idea of this function was to be concise, the new name kinda fails at that... If you don't like a function `MB`, maybe define it as a constant to multiply by instead?